torch/_inductor/codegen/wrapper.py

```diff
@@ -602,6 +616,9 @@ def append_precomputed_sizes_to_prefix(self):
                 )
 
     def codegen_python_sizevar(self, x: Expr) -> str:
+        # If sizevar is an expected data-dependent symbol, then just use its string repr.
+        if str(x) in V.graph.data_dep_symbol_to_expr_str:
+            return pexpr(x)
```

Comment:
Nyeh, this shouldn't be necessary. How does simplify choke on data-dependent symbol?